spec/javascripts/components/story/story_item_spec.js

```diff
@@ -80,4 +80,34 @@ describe('<StoryItem />', () => {
       expect(children).toHaveProp('title', title);
     });
   });
+
+  const render = props => {
+    const wrapper = shallow(<StoryItem {...props} />);
+    const expandedStory = wrapper.find('[data-id="expanded-story"]');
+    return { wrapper, expandedStory };
```

Comment:
Are you using `wrapper`? I could not find any reference

Reply:
Nop, I am not using, but I believe it's good keep this for future tests that will use wrapper.